fluss-server/src/main/java/com/alibaba/fluss/server/replica/ReplicaManager.java

```diff
@@ -1239,6 +1239,10 @@ private StopReplicaResultForBucket stopReplica(
             }
 
             remoteLogManager.stopReplica(replicaToDelete, delete && replicaToDelete.isLeader());
+            if (delete) {
```

Comment:
Only replica leader need to drop remote kv snapshot.... 
=> `if (delete && replicaToDelete.isLeader())`